modules/compliance-profile-types.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-concepts/compliance-operator-understanding.adoc
+// * security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc
+
+[id="compliance_profile_types_{context}"]
+== Compliance Operator profile types
+
+There are three types of compliance profiles available: Platform Node, and {op-system-first} profiles. 

```

Comment:
Better to be `There are three types of compliance profiles available: Platform, Node, and {op-system-first} profiles. `

Reply:
Good catch, thank you!